src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -667,6 +676,57 @@ public float AudioPitch
             }
         }
 
+        /// <summary>
+        /// Gets or sets the default codec type of the audio decoder.
+        /// </summary>
+        /// <value>A <see cref="CodecType"/> that specifies the type.
+        /// The default codec type could be different depending on the device capability.</value>
+        /// <remarks>
+        /// <para>To set, the player must be in the <see cref="PlayerState.Idle"/> state.</para>
+        /// <para><see cref="CodecType.Sw"/> is supported basically.
```

Comment:
Does this means software audio codec should be supported in all platform, right?

Reply:
as far as i understand it has to be but there still has small possibility that it isn't.
i got the vague answer so need to check again.

Reply:
The statement is strictly described as it's `supported basically`. 
So if not, you should describe it here.

Reply:
i just remembered this meaning.
`
The default codec type could be different depending on the device capability.
S/W codec type is supported basically.
`
these sentences are a set.
the default codec type can be hw or sw depending on the device. but sw is supported basically.

Reply:
`basically` might be put, in case sw codec is not supported